Maven build failed on Java 9

Trying to build Dubbo on Java 9, I got build failure with the following stack trace:

```
Caused by: java.lang.StringIndexOutOfBoundsException: begin 0, end 3, length 1
        at java.base/java.lang.String.checkBoundsBeginEnd(String.java:3116)
        at java.base/java.lang.String.substring(String.java:1885)
        at org.apache.commons.lang.SystemUtils.getJavaVersionAsFloat(SystemUtils.java:1133)
        at org.apache.commons.lang.SystemUtils.<clinit>(SystemUtils.java:818)
        ... 59 more
```

Looks like commons-lang does not understand the version semantics of Java 9.

--- a/dependencies-bom/pom.xml
+++ b/dependencies-bom/pom.xml
@@ -78,7 +78,7 @@
         <mina_version>1.1.7</mina_version>
         <grizzly_version>2.1.4</grizzly_version>
         <httpclient_version>4.5.3</httpclient_version>
-        <fastjson_version>1.2.31</fastjson_version>
+        <fastjson_version>1.2.46</fastjson_version>
         <zookeeper_version>3.4.9</zookeeper_version>
         <zkclient_version>0.2</zkclient_version>
         <curator_version>2.12.0</curator_version>

--- a/pom.xml
+++ b/pom.xml
@@ -68,7 +68,7 @@ limitations under the License.
         <maven_compiler_version>3.6.0</maven_compiler_version>
         <maven_source_version>3.0.1</maven_source_version>
         <maven_cobertura_version>2.7</maven_cobertura_version>
-        <maven_javadoc_version>2.10.4</maven_javadoc_version>
+        <maven_javadoc_version>3.0.0</maven_javadoc_version>
         <maven_jetty_version>6.1.26</maven_jetty_version>
     </properties>
 

--- a/pom.xml
+++ b/pom.xml
@@ -168,15 +168,6 @@ limitations under the License.
             </build>
         </profile>
         <profile>
-            <id>java8-doclint-disabled</id>
-            <activation>
-                <jdk>[1.8,)</jdk>
-            </activation>
-            <properties>
-                <javadoc.opts>-Xdoclint:none</javadoc.opts>
-            </properties>
-        </profile>
-        <profile>
             <id>java8-vm-args</id>
             <activation>
                 <jdk>[1.8,)</jdk>
@@ -229,7 +220,7 @@ limitations under the License.
                             <goal>jar</goal>
                         </goals>
                         <configuration>
-                            <additionalparam>${javadoc.opts}</additionalparam>
+                            <doclint>none</doclint>
                         </configuration>
                     </execution>
                 </executions>

